ts/Series/Treegraph/TreegraphNode.ts

```diff
@@ -0,0 +1,206 @@
+/* *
+ *
+ *  (c) 2010-2022 Pawel Lysy Grzegorz Blachlinski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ * Imports
+ *
+ * */
+
+
+import type TreegraphSeries from './TreegraphSeries.js';
+import type TreemapNodeNS from '../Treemap/TreemapNode.js';
+
+import TreegraphPoint from './TreegraphPoint.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+const {
+    seriesTypes: {
+        treemap: {
+            prototype: {
+                NodeClass: TreemapNode
+            }
+        }
+    }
+} = SeriesRegistry;
+
+interface TreegraphNode {
```

Comment:
- Interface for class prototype should come after class itself
- Section comments